docs/source/tutorials/qubo_problem.ipynb

```diff
@@ -0,0 +1,580 @@
+{
```

Comment:
<p>maybe add link to CVaR paper?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tencent-quantum-lab/tensorcircuit/pull/186/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>